tests/snuba/search/test_backend.py

```diff
@@ -121,7 +120,7 @@ def setUp(self):
             data={
                 "fingerprint": ["put-me-in-group1"],
                 "event_id": "a" * 32,
-                "message": "foo. Also, this message is intended to be greater than 256 characters so that we can put some unique string identifier after that point in the string. The purpose of this is in order to verify we are using snuba to search messages instead of Postgres (postgres truncates at 256 characters and clickhouse does not). santryrox.",
```

Comment:
I subtly adjusted the text here to avoid the wildcards used in the test.  most of them used `so*t` which is why all the words modified have `so` in them